Jenkinsfile

```diff
@@ -18,7 +18,7 @@ node('docker') {
         dockerCleanup()
         // hack
         dir('worker') {
```

Comment:
This was necessary when the repos were private as docker couldn't clone them during build. This quick fix will enable CI/CD again. We can drop the worker hack later, it won't do any harm now :)